bin/pipelineTiming.sh

```diff
@@ -210,20 +209,25 @@ fi
 logsDir="${TMP_ROOT_DIR}/logs/"
 projectStartDir="${TMP_ROOT_DIR}/logs/Timestamp"
 
-rsync -av ${sourceServerFQDN}:"${SCR_ROOT_DIR}/logs/Timestamp/*.csv" "${projectStartDir}" 
+rsync -av "${sourceServerFQDN}:${SCR_ROOT_DIR}/logs/Timestamp/*.csv" "${projectStartDir}" 
 
-for projectSheet in $(ls "${projectStartDir}/"*".csv")
+projectSheets=$(find "${projectStartDir}/" -name '*.csv' -type f)
+for projectSheet in "${projectSheets[@]}"
 do 
 	project=$(basename "${projectSheet}" .csv)
 
 	## determine run number.
-	if [ -e "${logsDir}/${project}/"*".generateScripts.finished" ]
+	
+	generateScriptsFinished=$(find "${logsDir}/${project}/" -name "*.generateScripts.finished")
+		
+	if [[ -e "${generateScriptsFinished}" ]]
```

Comment:
Deze ```find``` resulteert potentieel in meer dan één bestand en dat gaat niet werken met de test ```-e```

Reply:
is dat zo, want die find doe je toch specifiek voor dat project? of bedoel je dat we eigenlijk moeten checken op run01.generateScripts.finished, want dat lijkt me ook niet de bedoeling?

Reply:
Ja je doet dit voor een specifiek project, dus je verwacht nooit meer dan 1 zo'n bestand, maar die find werkt recursief en kan meerdere bestanden ophoesten. Zonder error handling crasht het dan op een leijke manier. De betere optie is: indien nul doe iets, indien 1 doe iets anders en indien 2 of meer genereer een foutmelding met wat er mis is.